angular/base/package.json

```diff
@@ -20,28 +20,28 @@
     "@angular/platform-browser": "5.2.9",
     "@angular/platform-browser-dynamic": "5.2.9",
     "@angular/router": "5.2.9",
-    "@ionic/angular": "0.2.1",
+    "@ionic/angular": "^4.0.0-alpha.1",
     "@ionic/schematics-angular": "latest",
     "core-js": "^2.5.3",
     "rxjs": "^5.5.8",
-    "zone.js": "^0.8.25"
+    "zone.js": "^0.8.26"
   },
   "devDependencies": {
-    "@angular/cli": "1.7.3",
+    "@angular-devkit/core": "^0.5.4",
+    "@angular-devkit/schematics": "^0.5.4",
+    "@angular/cli": "^1.7.4",
     "@angular/compiler": "5.2.9",
     "@angular/compiler-cli": "5.2.9",
-    "@angular-devkit/core": "0.3.2",
-    "@angular-devkit/schematics": "0.3.2",
     "@types/jasmine": "2.8.6",
     "codelyzer": "^4.0.1",
-    "jasmine-core": "~2.8.0",
+    "jasmine-core": "^3.1.0",
```

Comment:
This has been pinned to only allow patch versions. You've updated it before, but I had to revert. If angular uses jasmine 3, it can no longer run the tests because of an error. Please test `ng build`, `ng serve`, `ng e2e`, and `ng test` before updating dependencies.

Thanks!